test/e2e/scenarios/start/benefitType.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const paths = require('paths');
+
+Feature('Benefit Type');
+
+Before((I) => {
+    I.createTheSession();
+});
+
+After((I) => {
+    I.endTheSession();
+});
+
+Scenario('When I enter PIP, I am taken to the DWP Issuing office page', (I) => {
+
+    I.fillField('#benefitType', 'Personal Independence Payment (PIP)');
+    I.click('Continue');
+    I.seeInCurrentUrl(paths.compliance.dwpIssuingOffice);
+
+});
+
+Scenario('When I enter a non PIP benefit type, I am taken to the download form page', (I) => {
+
+    I.fillField('#benefitType', 'Disability Living Allowance (DLA)');
```

Comment:
Can we use this instead please: `I.enterBenefitTypeAndContinue('dla')`; or something similar. I think the majority of the time a user will select an option from the auto-complete. I can't see them typing `Disability Living Allowance (DLA)`.